src/index.ts

```diff
@@ -430,10 +433,11 @@ export default function oasToHar(
     }
   }
 
-  let requestBody: MediaTypeObject;
+  let requestBody: SchemaWrapper;
   if (operation.hasRequestBody()) {
-    // @ts-expect-error TODO `requestBody` coming back as `false | MediaTypeObject | [string, MediaTypeObject]` seems like a problem
-    [, requestBody] = operation.getRequestBody();
+    requestBody = operation
+      .getParametersAsJSONSchema()
+      .find(p => p.type === (operation.isFormUrlEncoded() ? 'formData' : 'body'));
```

Comment:
`formData` is used in our API Explorer for `application/x-www-form-urlencoded` endpoints and if you have an operation with that, it will only ever have a `formData`. `body` is used for all other payload shapes.

Reply:
sure